### PR TITLE
feat(presets): Add nenoNaninu/TypedSignalR.Client

### PR DESCRIPTION
**Pull request by sommmen**
## Changes

Adds https://github.com/nenoNaninu/TypedSignalR.Client

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — COMMENTED** [review]
Do these repos all release synchronized versions at the same time?

**Review by viceice — DISMISSED** [review]

**Comment by sommmen** [comment]
> Do these repos all release synchronized versions at the same time?

I'm not sure exactly what you're asking - but i don't think so. As in, if packages in the main repo are updated, this does not automatically mean a release of .DevTools and/or .Typescript, the author seems to keep things seperate in that regard. I can imagine if there are breaking changes all packages of all git repos would be updated.

Would you like me to remove those and then simply refer to the main repo?

File: lib/config/presets/internal/monorepo.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/monorepo.ts:495]: > > Do these repos all release synchronized versions at the same time?
> 
> I'm not sure exactly what you're asking - but i don't think so. As in, if packages in the main repo are updated, this does not automatically mean a release of .DevTools and/or .Typescript, the author seems to keep things seperate in that regard. I can imagine if there are breaking changes all packages of all git repos would be updated.
> 
> Would you like me to remove those and then simply refer to the main repo?

Commenting this as a review comment so we don't accidentally merge this before it's ready. I'll let the maintainers decide what to do.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Comment by rarkins** [comment]
> > Do these repos all release synchronized versions at the same time?
> 
> I'm not sure exactly what you're asking - but i don't think so. As in, if packages in the main repo are updated, this does not automatically mean a release of .DevTools and/or .Typescript, the author seems to keep things seperate in that regard. I can imagine if there are breaking changes all packages of all git repos would be updated.
> 
> Would you like me to remove those and then simply refer to the main repo?

If they are from separate repos, and they don't have the same/synchronized versions, then it's not qualified for our "monorepo" grouping. Instead it could be part of group:recommended if you think people usually should update them together at once

**Comment by sommmen** [comment]
> > > Do these repos all release synchronized versions at the same time?
> > 
> > 
> > I'm not sure exactly what you're asking - but i don't think so. As in, if packages in the main repo are updated, this does not automatically mean a release of .DevTools and/or .Typescript, the author seems to keep things seperate in that regard. I can imagine if there are breaking changes all packages of all git repos would be updated.
> > Would you like me to remove those and then simply refer to the main repo?
> 
> If they are from separate repos, and they don't have the same/synchronized versions, then it's not qualified for our "monorepo" grouping. Instead it could be part of group:recommended if you think people usually should update them together at once

I think just having the one monorepo in there is the way to go about things. Updated the PR, hopefully to satisfaction this time. Apologies for the initial wrongful pr.

File: lib/config/presets/internal/monorepo.ts
Comment (rarkins) [inline lib/config/presets/internal/monorepo.ts:495]: Should it have client in the name?

Comment (sommmen) [reply lib/config/presets/internal/monorepo.ts:495]: I don't think you need that distinction, the SignalR server is already typed OOB. I have no issue with adding '-client' at the end if you want it to be extra verbose. Let me know what you think.

**Review by rarkins — APPROVED** [review]

**Pull request merged**
